R-packages/covidcast/R/covidcast.R

```diff
@@ -494,11 +494,16 @@ covidcast_days <- function(data_source, signal, start_day, end_day, geo_type,
                           as_of = as_of,
                           issues = issues,
                           lag = lag)
-    message(sprintf("Fetched day %s: %s, %s, num_entries = %s",
-                    query_day,
-                    dat[[i]]$result,
-                    dat[[i]]$message,
-                    nrow(dat[[i]]$epidata)))
+    summary <- sprintf(
+      "Fetched day %s: %s, %s, num_entries = %s",
+      query_day,
+      dat[[i]]$result,
+      dat[[i]]$message,
+      nrow(dat[[i]]$epidata)
+    )
+    if (length(summary) != 0) {
```

Comment:
Is it only the case when `message != "success"` that we have NULLs? If so, we can just move this `message()` down into the `if (dat[[i]]$message == "success")` block just below, and not need this separate length check.

Reply:
For this specific case that looks true (message = "no results"). I did it this way since the same bug could be triggered by any of the parameters being null (or the message being changed to include different, null paramters), so this fixes the general case of empty message = spurious newlines. If you feel that those are unlikely and prefer the simpler move into the success block, I'm fine with that too. wdyt?

Reply:
I suppose we can be defensive and keep it as-is, in case of weird things (like Epidata changing how it returns data in specific error cases)